packages/react-client/src/__tests__/ReactFlight-test.js

```diff
@@ -1241,10 +1241,10 @@ describe('ReactFlight', () => {
     const ClientErrorBoundary = clientReference(MyErrorBoundary);
 
     function App() {
-      return (
-        <ClientErrorBoundary>
-          <ServerComponent />
-        </ClientErrorBoundary>
```

Comment:
Since this executes on the server it needs to use ReactServer.

It didn't have the correct owner before.